stacks_queues/stack.py

```diff
@@ -12,7 +12,8 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        
+        return self.store.add_first(element)
```

Comment:
Per the specification, the function should return `None`
```suggestion
        self.store.add_first(element)
```